README.md

```diff
@@ -60,4 +60,145 @@ A Venue object contains information about various venues where Events are held.
 }
 ```
 
+---
+
 ## Sample requests and responses
+This is an example of a request to the `/submit` endpoint, which expects an event details object and either the ID of an existing venue or venue object if it doesn't exist yet. 
+
+The response data will be a JSON object with a status attribute and both the newly created event and venue.
+
+### Request 1: Creating new event with existing venue
+
+```json
+// POST https://what-is-good-manitoba.ca/submit
+{
+    "details": {
+        "ID": 945,
+        "ticket": "https://ticketmb.com/event/ted-x-mb-2023",
+        "host": "TEDxManitoba",
+        "description": "Enjoy Ted Talks from this independently organized TED event",
+        "startTime": "2023/04/30/17",
+        "endTime": "2023/04/30/19",
+        "tags": ["talk", "documentary"],
+    },
+    "venue": {
```

Comment:
For this, since the venue already exists I think they should only need to specify the venue ID, so rather than the full venue object here they would only need to do: 
``` js
  "venue": 5056
```